.github/workflows/developer_onboarding_notification.yml

```
@@ -0,0 +1,73 @@
+
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
+
+jobs:
+  comment-on-merge:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
```

```suggestion
if: github.event.pull_request.merged == true
```

Since the workflow is already filtered to run only on the develop branch, the `if` condition now only checks if the PR is merged:

---

My apologies. My initial idea was to make a change so that we have:

```yml
on:
  pull_request_target:
    types: [closed]
    branches:
      - develop
```

But after researching I have found out that the `pull_request_target` event does not support filtering based on branches directly under branches. Instead, to filter based on branches, you should use `pull_request` with a base branch condition, so your original code was correct. Please revert that line to what you had originally.